dags/burnham.py

```diff
@@ -518,6 +519,30 @@ def encode_test_scenarios(test_scenarios):
     return b64_encoded
 
 
+def do_sleep(minutes):
+    """Sleep for the given number of minutes.
+
+    Writes out an update every minute to give some indication of aliveness.
+    """
+    print(f"Configured to sleep for {minutes} minutes. Let's begin.")
+    for i in range(minutes, 0, -1):
+        print(f"{i} minute(s) of sleeping left")
```

Comment:
Do you generally recommend `print` commands over logging in Airflow operators?

Reply:
Logging rather printing is a very good idea 👍